geometry_msgs/PoseArray data
float32 duration
---
int16 result